app/views/admin/conference/_recent_users.html.haml

```diff
@@ -6,18 +6,28 @@
           %th #
           %th E-Mail
           %th Date registered
-          %th Status
+          %th Confirmed?
       - recent_users.each_with_index do |user, index|
         %tbody
           %tr
             %td= index + 1
             %td= link_to user.email, admin_user_path(user.id)
             %td= user.created_at.strftime('%m/%d/%Y')
             %td
-              - if user.confirmed?
-                %span.label.label-success Confirmed
-              -else
-                %span.label.label-danger Unconfirmed
+              - if can? :toggle_confirmation, @user
+                = check_box_tag user.id, user.id, user.confirmed?, 
```

Comment:
I would make this checkbox a disabled one. This page is not where people would usually care to change data. It can be done from admin/users#index and admin/users#show, that sounds good enough to me.